test/components/NewFilterTest.js

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert')
+const React = require('react')
+const ReactDOM = require('react-dom')
+const TestUtils = require('react-addons-test-utils')
+const NewFilter = require('../../src/components/NewFilter/NewFilter.jsx')
```

Comment:
You should be able to just `require('../../src/components/NewFilter')` if that directory has a package.json file in it like the other package.jsons (excluding the root one).